media/js/firefox/product-page.js

```diff
@@ -0,0 +1,32 @@
+(function($, Waypoint) {
```

Comment:
This file seems identical to `sub-nav.js` below - suggest removing this file and using the common shared file?